tpp-app/App.js

```diff
@@ -7,16 +7,14 @@ import Calendar from './src/home/Calendar';
 import Info from './src/info/Info';
 import Settings from './src/settings/Settings';
 
-import InfoIcon from './ios/tppapp/Images.xcassets/info-icon-3x.png'
-import BloodDropIcon from './ios/tppapp/Images.xcassets/icons/blood-drop.png'
-import CalendarIcon from './ios/tppapp/Images.xcassets/icons/calendar-icon.png'
-import SettingsIcon from './ios/tppapp/Images.xcassets/icons/settings_icon.png';
+import VectorImage from 'react-native-vector-image';
 
 const Tab = createBottomTabNavigator();
 
 const CustomTabBarButton = ({ onPress }) => {
   const calendarShowing = useIsFocused();
-  let icon = calendarShowing ? BloodDropIcon : CalendarIcon;
+  
+  let icon = calendarShowing ? <VectorImage source={require('./ios/tppapp/Images.xcassets/icons/blood_drop.svg')} /> : <VectorImage source={require('./ios/tppapp/Images.xcassets/icons/calendar_icon.svg')} />;
```

Comment:
Moves around the logic a bit so that it's shorter
```suggestion
  let icon = calendarShowing ? './ios/tppapp/Images.xcassets/icons/blood_drop.svg' : './ios/tppapp/Images.xcassets/icons/calendar_icon.svg';
```